be/src/olap/merger.cpp

```diff
@@ -193,6 +193,10 @@ Status Merger::vertical_compact_one_group(
     }
     reader_params.tablet_schema = merge_tablet_schema;
 
+    if (tablet->enable_unique_key_merge_on_write()) {
```

Comment:
We should be able to cover this issue in UT and regression test?

Reply:
add it in UT.